testing/scenario_app/pubspec.yaml

```diff
@@ -15,6 +15,9 @@ environment:
 # check the .packages and .package_config to make sure all the paths are
 # relative to this directory into //third_party/dart, or //third_party/pkg
 dependencies:
+  args: 2.1.0
+  path: 1.8.0
+  process: 4.2.3
```

Comment:
These need ot be pullable from the SDK - we cannot use dependencies from pub in engine CI. That's why the dependency overrides below are used.